packages/socket-stream-client/browser-legacy.js

```diff
@@ -0,0 +1,5 @@
+// Don't import SockJS for legacy or unknown browsers that supports WebSockets
+if (!(Package['disable-sockjs']
+  && typeof WebSocket !== 'undefined')) {
+  import("./sockjs-0.3.4.js");
```

Comment:
You'll need to add `dynamic-import` as a dependency in `package.js` again.
It was removed recently so `unsafe-eval` could be disabled via CSP for folks that needed to

Reply:
@coagmano thanks for watching.

The `ecmascript` package implements the `dynamic-import`, see:

`packages/ecmascript/package.js`

```js
  // Runtime support for Meteor 1.5 dynamic import(...) syntax.
  api.imply('dynamic-import');
```

Reply:
Oooh, I didn't know that.
That means [this PR of mine](https://github.com/meteor/meteor/pull/10741/) did nothing 👀